src/views/backoffice/BackofficeProductUpdate.vue

```diff
@@ -67,6 +67,7 @@ const updateItem = async () => {
       .then(() => {
         showToast('success', 'Produkt erfolgreich aktualisiert')
         itemsStore.getItems()
+        router.push({ name: 'Backoffice Product Settings' })
```

Comment:
I receive a 400-Error that products could not be updated. After commenting this line, everything works fine.
Therefore, I assume this line should be executed below this try-catch statement